docs/UserGuide.md

```diff
@@ -105,14 +105,14 @@ Below is a sample unsuccessful command executed. Note that the command is highli
   * All input values are case-sensitive unless otherwise specified.
 
 * Duplicate contacts:
-  * Contacts are considered duplicates if they share the same phone number.
+  * Contacts are considered duplicates if they share the same mobile phone number and is used as unique identifier of a Person as no two persons would share the same mobile phone number.
```

Comment:
```suggestion
* In FINDvisor, contacts are **uniquely identified** using their mobile phone number. Hence, two contacts are considered duplicates if they share the same mobile phone number. 
```
@KohGuanZeh @YuZizhen Just a nit, but does this sound better?

Reply:
Looks good to me. I think we should adjust it to this.